res/values-ua/strings.xml

```diff
@@ -0,0 +1,114 @@
+	<string name="dictionary_load_bad_char">Завантаження не вдалося. Невірне слово \"%1$s\" у рядку %2$d мови \"%3$s\".</string>
+	<string name="dictionary_load_error">Не вдалося завантажити словник для мови \"%1$s\" (%2$s).</string>
+	<string name="dictionary_load_cancelled">Завантаження словника скасовано.</string>
+	<string name="dictionary_loaded">Словник завантажено.</string>
+	<string name="dictionary_loading">Словник завантажується (%1$s)…</string>
+	<string name="dictionary_loading_indeterminate">Словник завантажується</string>
+	<string name="dictionary_load_title">Завантажити Словник</string>
+	<string name="dictionary_missing_go_load_it">Нема словника для мови \"%1$s\". Перейдіть в Налаштування та завантажте його.</string>
+	<string name="dictionary_not_found">Помилка завантаження. Словник для мови \"%1$s\" не знайдений.</string>
+	<string name="dictionary_truncate_title">Видалити все</string>
+	<string name="dictionary_truncate_unselected">Видалити невибрані</string>
+	<string name="dictionary_truncated">Словник видалено.</string>
+	<string name="dictionary_truncating">Видаляється…</string>
+
+	<string name="donate_title">Підтримати</string>
+	<string name="donate_summary">Якщо вам подобається %1$s, ви можете підримати розробку по посиланню: %2$s.</string>
+	<string name="donate_url" translatable="false">https://www.buymeacoffee.com/sspanak</string>
+	<string name="donate_url_short" translatable="false">www.buymeacoffee.com</string>
+
+	<string name="function_add_word_key">Додати слово</string>
+	<string name="function_backspace_key">Стерти</string>
+	<string name="function_change_keyboard_key">Змінити клавіатуру</string>
+	<string name="function_filter_clear_key">Очистити фільтр</string>
+	<string name="function_filter_suggestions_key">Фільтрувати пропозиції</string>
+	<string name="function_previous_suggestion_key">Попередня пропозиція</string>
+	<string name="function_next_suggestion_key">Наступна пропозиція</string>
+	<string name="function_next_language_key">Наступна мова</string>
+	<string name="function_next_mode_key">Режим вводу</string>
+	<string name="function_show_settings_key">Показати налаштування</string>
+	<string name="function_reset_keys_title">Відновити стандартні клавіші</string>
+	<string name="function_reset_keys_done">Стандартні налаштування клавіш відновленно.</string>
+
+	<string name="setup_keyboard_status">Статус</string>
+	<string name="setup_default_keyboard">Вибрати стандартну клавіатуру</string>
+	<string name="setup_tt9_on">Traditional T9 увімкнено</string>
+	<string name="setup_tt9_off">Traditional T9 вимкнуто</string>
+	<string name="setup_click_here_to_enable">Нажміть тут, щоб увімкнути TT9 в налаштуваннях Android.</string>
+
+	<string name="key_hold_key">(тримати)</string>
+	<string name="key_none" translatable="false">--</string>
+	<string name="key_back">Назад</string>
+	<string name="key_call">Подзвонити</string>
+	<string name="key_dpad_up" translatable="false">▲</string>
```

Comment:
Please, remove the non-translatable strings. TT9 will read them from the base English strings.xml. The special characters are the same in all languages, so it makes no sense to duplicate them.

Reply:
I just took English xml and translated in-place
Will clean up untranslatable stuff and there's less of it now (: